src/endpoints.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * This file registers REST API Endpoints.
+ *
+ * @package wp-conditional-blocks
+ */
+
+namespace Alley\WP\WP_Conditional_Blocks;
+
+/**
+ * Registers endpoints.
+ *
+ * @return void
+ */
+function register_endpoints(): void {
+	new Endpoint_Get_Conditions();
```

Comment:
Might we be able to standardize around how we're adding features in `wp-alleyvate` using the `Feature` class available in https://github.com/alleyinteractive/wp-type-extensions?

Reply:
I believe I implemented the `Feature` class here.